Docs: review code blocks with new metadata (#2637)
keystonejs/keystone: .changeset/metal-pigs-enjoy.md

---
'@keystonejs/website': patch
---

> Second pass after #2631 to cleanup any weirdness that the new code block metadata introduced.

Author: ```suggestion
Cleanup any weirdness that the new code block metadata introduced in #2631.
```
